### PR TITLE
task/399-create-bubble-for-backup-screen

### PR DESCRIPTION
**Pull request by ciripel**

### CODE REVIEW

**Review by JiriBestak — APPROVED** [review]

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart [hunk 1]
```diff
@@ -38,175 +38,181 @@ class _BackupInventoryScreen extends State<BackupInventoryScreen> {
...
-                        style: SioTextStyles.h3.apply(
-                          color: SioColors.whiteBlue,
-                        ),
+                      ),
+                    ],
+                  ),
+                  const SizedBox(
+                    height: Dimensions.padding14,
+                  ),
+                  Align(
```
Comment (storkandstars) [inline lib/view/screens/authenticated_screens/backup_inventory_screen.dart:77]: Row and Column have main and cross axis alignment built in.

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart [hunk 2]
```diff
@@ -38,175 +38,181 @@ class _BackupInventoryScreen extends State<BackupInventoryScreen> {
...
+                            child: HighlightedElevatedButton.disabled(
+                              label: context
+                                  .locale.backup_inventory_screen_button1_label,
+                            ),
+                          )
+                        : SizedBox(
+                            width: double.infinity,
+                            child: HighlightedElevatedButton.primary(
+                              onPressed: () {
+                                setState(() {
```
Comment (storkandstars) [inline lib/view/screens/authenticated_screens/backup_inventory_screen.dart:186]: We do not set local state here.

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart [hunk 3]
```diff
@@ -38,175 +38,181 @@ class _BackupInventoryScreen extends State<BackupInventoryScreen> {
...
+                        child: HighlightedElevatedButton.disabled(
+                          rightIcon: SioIcons.done,
+                          label: context
+                              .locale.backup_inventory_screen_button2_label,
+                        ),
+                      ),
+              ],
+            ),
+          ),
+          const SizedBox(
```
Comment (storkandstars) [inline lib/view/screens/authenticated_screens/backup_inventory_screen.dart:212]: We might to replace these gaps within Gaps constants where we would define them e.g.`Gaps.vertical20`

**Review by storkandstars — CHANGES_REQUESTED** [review]

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart (hunk 1 above)
Comment (ciripel) [reply lib/view/screens/authenticated_screens/backup_inventory_screen.dart]: fixed in https://github.com/SimplioOfficial/simplio-app/pull/536/commits/301c4d2cb601ecb1094f9542b0d56b9867c3e3f1

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart (hunk 2 above)
Comment (ciripel) [reply lib/view/screens/authenticated_screens/backup_inventory_screen.dart]: fixed in https://github.com/SimplioOfficial/simplio-app/pull/536/commits/301c4d2cb601ecb1094f9542b0d56b9867c3e3f1

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart (hunk 3 above)
Comment (ciripel) [reply lib/view/screens/authenticated_screens/backup_inventory_screen.dart]: fixed in https://github.com/SimplioOfficial/simplio-app/pull/536/commits/dd20181ab4fad37359c9811cea159e903cedd38d

**Pull request merged**